src/formatDate.js

```diff
@@ -50,7 +50,27 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const objectDate = {};
+  const separatorToFormat = toFormat[toFormat.length - 1];
+  const separatorFromFormat = fromFormat[toFormat.length - 1];
+  const arrayDate = date.split(separatorFromFormat);
+  const result = [];
+
+  for (let i = 0; i < arrayDate.length; i++) {
```

Comment:
Consider using `arrayDate.forEach` instead of a for loop to fill `objectDate`. This is more readable and avoids potential off-by-one errors.